### PR TITLE
Add ability to ask inspection

### PR DESCRIPTION
**Pull request by motim6026**

### CODE REVIEW

File: app/src/main/java/in/avimarine/boatangels/BoatHolder.java [hunk 1]
```diff
@@ -31,6 +33,9 @@ public BoatHolder(View itemView) {
   public void bind(Boat b) {
     setName(b.name);
     //DateUtils.getRelativeTimeSpanString(b.lastInspectionDate)
+    String SumOffer = Integer.toString(b.offerPoint);
```
Comment (aayaffe) [inline app/src/main/java/in/avimarine/boatangels/BoatHolder.java]: You continue to use capital letters for local variables

File: app/src/main/java/in/avimarine/boatangels/BoatHolder.java [hunk 2]
```diff
@@ -17,12 +17,14 @@
 
 public class BoatHolder extends RecyclerView.ViewHolder {
   private final TextView mNameField;
+  private final TextView mSumOfferPoints;
```
Comment (aayaffe) [inline app/src/main/java/in/avimarine/boatangels/BoatHolder.java]: The letter m in the beginning of the variable's name is useless.
This is the style guide we use: https://google.github.io/styleguide/javaguide.html#s5.1-identifier-names
In particular: 
In Google Style special prefixes or suffixes, like those seen in the examples name_, mName, s_name and kName, are **not** used.

File: app/src/main/java/in/avimarine/boatangels/activities/AskInspectionActivity.java [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+  private String Uid;
+  private String points;
+  private final String  p = " points";
+
+
+  @Override
+  protected void onCreate(Bundle savedInstanceState) {
+    super.onCreate(savedInstanceState);
+    setContentView(R.layout.activity_ask_for_inspection);
+    DocumentReference docRef = db.collection("users").document(FirebaseAuth.getInstance().getUid());
```
Comment (aayaffe) [inline app/src/main/java/in/avimarine/boatangels/activities/AskInspectionActivity.java]: Why don't you use the FireBase class?
Accessing the DB directly without using the class designed for it is bad practice.

File: app/src/main/java/in/avimarine/boatangels/db/objects/Boat.java [hunk 4]
```diff
@@ -18,20 +18,76 @@
   public GeoPoint location;
   public String marinaUuid;
   public String marinaName;
+  public int offerPoint = 0;
+  public int yachtiePoint = 100;
```
Comment (aayaffe) [inline app/src/main/java/in/avimarine/boatangels/db/objects/Boat.java]: The user is the one that has the points not the boat.

File: app/src/main/java/in/avimarine/boatangels/activities/BoatForInspectionActivity.java [hunk 5]
```diff
@@ -38,7 +38,8 @@ protected void onCreate(Bundle savedInstanceState) {
     ButterKnife.bind(this);
     Query query = FirebaseFirestore.getInstance()
         .collection("boats")
-//        .orderBy("lastInspectionTime")
+        .whereGreaterThan("offerPoint", 0)
```
Comment (aayaffe) [inline app/src/main/java/in/avimarine/boatangels/activities/BoatForInspectionActivity.java]: This means that any boat that did not offer points is not shown in the list.
This is not the use case we discussed

File: app/src/main/java/in/avimarine/boatangels/db/objects/Inspection.java [hunk 6]
```diff
@@ -20,11 +20,13 @@
   public Map<String, String> finding;
   public Long inspectionTime;
   public List<Message> discussion;
+  public int getPoint;
```
Comment (aayaffe) [inline app/src/main/java/in/avimarine/boatangels/db/objects/Inspection.java]: Not a very good name.
A better name would just be pointsEarned for instance

File: app/src/main/res/layout/activity_ask_for_inspection.xml [hunk 7]
```diff
@@ -0,0 +1,93 @@
...
+
+    android:textColor="#000000"
+    android:textSize="22dp"
+    app:layout_constraintBottom_toBottomOf="parent"
+    app:layout_constraintEnd_toEndOf="parent"
+    app:layout_constraintHorizontal_bias="0.41"
+
+    app:layout_constraintStart_toStartOf="parent"
+    app:layout_constraintTop_toTopOf="parent"
+    app:layout_constraintVertical_bias="0.296"/><![CDATA[
```
Comment (aayaffe) [inline app/src/main/res/layout/activity_ask_for_inspection.xml]: What is that?

File: app/src/main/java/in/avimarine/boatangels/activities/AskInspectionActivity.java [hunk 8]
```diff
@@ -0,0 +1,109 @@
...
+        Uid = getBoatUid.replaceAll("[]\\[]", "");
+        showPoint(Uid);
+
+       }
+    });
+  }
+
+
+  @OnClick(R.id.ask_inspection_btn)
+  public void inspectMe(View v) {
```
Comment (aayaffe) [inline app/src/main/java/in/avimarine/boatangels/activities/AskInspectionActivity.java]: You forgot finish() at the end...

File: app/src/main/res/layout/activity_ask_for_inspection.xml [hunk 9]
```diff
@@ -0,0 +1,93 @@
...
+  android:layout_height="match_parent"
+  tools:context="in.avimarine.boatangels.activities.AskInspectionActivity">
+
+  <TextView
+    android:id="@+id/ask_for_inspection_title"
+    android:layout_width="wrap_content"
+    android:layout_height="wrap_content"
+    android:text="@string/ask_for_inspection_title"
+    android:textColor="#000000"
+    android:textSize="24dp"
```
Comment (aayaffe) [inline app/src/main/res/layout/activity_ask_for_inspection.xml]: textSize should use sp and not dp:
https://stackoverflow.com/questions/23981260/should-use-sp-instead-of-dp-for-text-sizes

File: app/src/main/res/layout/activity_ask_for_inspection.xml (hunk 7 above)
Comment (motim6026) [reply app/src/main/res/layout/activity_ask_for_inspection.xml]: i fixed it

**Pull request closed without merging**